Dockerfile

```diff
@@ -1,14 +1,40 @@
-FROM gradle:jdk8 as builder
+FROM gradle:jdk8 as cache
+RUN mkdir -p /home/gradle/cache
+ENV GRADLE_USER_HOME /home/gradle/cache
+COPY build.gradle /home/gradle/src/
+WORKDIR /home/gradle/src
+RUN gradle clean dependencies -i --stacktrace
 
+FROM gradle:jdk8 AS builder
+COPY --from=cache /home/gradle/cache /home/gradle/.gradle
 COPY --chown=gradle:gradle . /home/gradle/src
+
 WORKDIR /home/gradle/src
-RUN set https_proxy=https://127.0.0.1:8001
-RUN set http_proxy=https://127.0.0.1:8001
-RUN gradle build
+RUN gradle clean build
+RUN mkdir  -p target
+WORKDIR /home/gradle/src/target
+RUN jar -xf ../build/libs/*.jar
+
+
 
 FROM openjdk:8-jre-slim
-COPY --from=builder /home/gradle/src/ckmirror/build/distributions/ckmirror.tar /app/
+COPY --from=builder /home/gradle/src/build/distributions/ckmirror.tar /app/
 WORKDIR /app
+RUN ls -al
 RUN tar -xvf ckmirror.tar
 WORKDIR /app/ckmirror
-CMD bin/ckmirror
\ No newline at end of file
+
+RUN sed -i 's/deb.debian.org/mirrors.aliyun.com/g' /etc/apt/sources.list
+RUN apt-get update && apt-get install -y cron && apt-get install -y vim
+
+RUN touch /var/spool/cron/crontabs/root
+RUN chmod 600 /var/spool/cron/crontabs/root
+RUN echo "EDITOR=vi; EXPORT EDITOR" >> /root/.profile
```

Comment:
I think this line is not necessary for the image.
Like the `vim` installation, you can set it in the running container.